src/Components/Reports/Util.js

```diff
@@ -135,3 +135,11 @@ export const formatExecutiveReportData = (data) => {
 
     return { stateChartData, stateTableData, conditionsChartData, conditionsTableData, condtionsInfo };
 };
+
+export const pluralize = (count, singular, plural) => {
+    if (!plural) {
```

Comment:
This looks better, just a thought, should we use https://github.com/plurals/pluralize ?
I am not sure how easy it is to add a dependency for the project, but this looks like not that complex. If you feel I am dragging the changes too much for Best possible solution then we can aim for this later.

Reply:
@upadhyeammit nice find and we can surely use the same. However for small use-cases I prefer adding own methods as it gives more control in case we want to update the logic. In future we can definitely discuss around when/when not to add libraries based on the scenario. Let me know if it make sense 🙌🏻

Reply:
Agree to what you said! yeah so from my side it looks good! Thanks for resolving all the suggestions!

Reply:
great thank you @upadhyeammit 💯